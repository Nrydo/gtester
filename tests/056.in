5614267
244991763
257946698