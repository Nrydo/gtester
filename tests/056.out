6029701314814049